Sources/SwiftCrossUI/Backend/AppBackend.swift

```diff
@@ -603,6 +604,91 @@ public protocol AppBackend: Sendable {
     /// ``showAlert(_:window:responseHandler:)``.
     func dismissAlert(_ alert: Alert, window: Window?)
 
+    /// Creates a sheet object (without showing it yet). Sheets contain View Content.
+    /// They optionally execute provied code on dismiss and
```

Comment:
Make `View Content` lowercase (unless it links to a type).

Replace `provied` with `provided`.